Dockerfile

```diff
@@ -33,7 +33,8 @@ RUN set -x \
  && /fossology/utils/fo-installdeps -e -y \
  && $_cleanup
 RUN curl -sS https://getcomposer.org/installer | php && \
-    mv composer.phar /usr/local/bin/composer
+	mkdir /tmp/composer && \
+    mv composer.phar /tmp/composer/composer
```

Comment:
The script `fo_installdeps` still installs composer to the "old" place: https://github.com/fossology/fossology/blob/35d95766b6000acefae7b9849f4718f0380c7acf/utils/fo-installdeps#L267